mage_ai/data_preparation/templates/data_loaders/redshift.py

```diff
@@ -0,0 +1,23 @@
+from mage_ai.data_loader.redshift import Redshift
+from pandas import DataFrame
+
+
+def load_data_from_redshift() -> DataFrame:
+    """
+    Template code for loading data from Redshift cluster.
+
+    This template assumes that IAM profiles are specified in `~/.aws`.
+    - If IAM profiles are not setup, manually specify them as keyword arguments in `config`
+    - If using temporary database credentials, use `Redshift.with_temporary_credentials()`
+    to create loader.
+    """
+    config = {
```

Comment:
we also need to provide host and port, right? what configs are stored in IAM profiles?

Reply:
I think the most common config is to provide the following configs: database, host, password, port, user